### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Nakerakantianjali**

### CODE REVIEW

File: Assignment1.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*Create a java program to search through the home directory and look for files that match a regular expression. The program should be able to take inputs repeatedly and should print out the full absolute path of the matching files found.*/
+
+import java.io.*;
+import org.apache.commons.io.filefilter.*;
+
+public class Main {
+
+    public static  void main(String[] args) throws IOException {
+	    //get home directory		
+            String str=System.getProperty("user.home");
```
Comment (narenraj701) [inline Assignment1.java]: Give meaningful name to the variables, like homeDirPath

File: Assignment10.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import java.util.LinkedList;
+
+public class SListiterator<T> {
+    Slist<T>root=null;
+    public void insert(Slist<T> node){
+        if(root==null) {
+            System.out.println("nullify");
+            node.next = root;
+            root = node;
+        }
```
Comment (narenraj701) [inline Assignment10.java:31]: Please write the complete logic that handles all the cases

File: Assignment11.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Using TextFile and a Map<Character,Integer>, create a program that takes the file name as a command line argument and counts the occurrence of all the different characters. Save the results in a text file.
+
+*/
+import java.io.*;
+
+Class Assignment11{
+	public static void main(String [] args)thorws Exception{
+		File file=new File(args[0]);
+ 		FileWriter fw=new FileWriter(file.getPath(),true);
+            	BufferedWriter bw=new BufferedWriter(fw);
```
Comment (narenraj701) [inline Assignment11.java]: Execute and check the outputs before submitting the Assignments
There is no map created here in this case and without creating you are accessing it

File: Assignment2.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        for(int i=0;i<str.length();i++){
+            c[str.charAt(i)-97]++;
+        }
+        for(int i=0;i<26;i++){
+            if(c[i]==0){
+                flag=0;
+		break;
+            }
+        }
+        if(flag==1 && str.length()>=26){
```
Comment (narenraj701) [inline Assignment2.java:29]: you could have checked the length in the starting to save Time

File: Assignment3.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        inetAddress=InetAddress.getByName(input);
+	//checking given host is reachable or not
+        if(inetAddress.isReachable(5000)){
+            stop=new Date();
+            System.out.println("median time taken to "+(stop.getTime()-start.getTime())/2);
+        }
+	else{
+		System.out.println(" not reachable");
+	}
+}
```
Comment (narenraj701) [inline Assignment3.java:26]: execute and submit only it is successfully executed

File: Assignment4.java [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+            return sdf.format(c4.getTime())+" "+sdf.format(c3.getTime());
+        }
+        else{
+            return sdf.format(c4.getTime())+" "+sdf.format(c2.getTime());
+        }
+        
+    }
+    Public Static void main(String[] args)throws IOException{
+	Assignment4 ass=new Assignment4();
+	BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
```
Comment (narenraj701) [inline Assignment4.java:84]: some lines are missing

**Comment by Nakerakantianjali** [comment]
closing pull request

**Pull request closed without merging**